### DISCUSSION TITLE
Problems when running build.cmd

### DISCUSSION THREAD

**Discussion by yuanyuan-lalala** [microsoft/AirSim#4674] [General]
When I using build.cmd in windows,there are some error.
![image](https://user-images.githubusercontent.com/73372324/186798257-7cde7e77-3253-49f1-bcc6-5a70d9e670a7.png)

**Comment by Yuhandswash** [comment]
Hello, have you solved this problem yet?

**Comment by heyBryceZhang** [comment]
I have the same problem.  I added "echo finished  " at the last line of "build.cmd",  it was been printed,  so I think this question has no effect on running "build.cmd".
